hack/generate.sh

```diff
@@ -13,21 +13,21 @@ function lint_swagger() {
 
 function generate_go_server() {
     rm -rf restapi
-    docker run -u $(id -u):$(id -u) -v ${__root}:${__root}:rw,Z -v /etc/passwd:/etc/passwd -w ${__root} \
+    docker run $UID_FLAGS -v ${__root}:${__root}:rw,Z -v /etc/passwd:/etc/passwd -w ${__root} \
```

Comment:
I think we have another usage of this file outside of this makefile, so it will be a good idea to provide some default values in case ``UID_FLAGS`` is not defined

Reply:
that's a very good point! Thanks, @osherdp !